proposal/embedded-protocol-2.md

```diff
@@ -53,6 +53,46 @@ This proposal makes three breaking changes to the embedded Sass protocol:
 * Move `CompileSuccess.loaded_urls` to `CompileResult.loaded_urls` so it's
   available even when compilation fails.
 
+### Design Decisions
+
+#### Length Before Compilation ID
+
+This proposal includes the compilation ID for each request _after_ the length,
+and includes the size of the compilation ID in the total length.
```

Comment:
Nit:  although it makes sense, I found the use of "includes" slightly confusing, perhaps it's my lack of familiarity with it in this context. I wonder if a different word such as "declares" or "defines" may make this clearer 🤔